include/revng/ADT/SortedVector.h

```diff
@@ -83,6 +83,15 @@ class SortedVector {
   }
 
   bool operator==(const SortedVector &) const = default;
+  std::strong_ordering operator<=>(const SortedVector &Another) const {
```

Comment:
Why do we need this?
Are we using this as a key in some `std::map`?

Reply:
In `std::set`, by your suggestion.

Reply:
https://github.com/revng/revng/pull/485#discussion_r2109326743